library/hawkular_alerts_group_member.py

```diff
@@ -210,7 +224,9 @@ def main():
             group_id=dict(required=True, type='str'),
             id=dict(required=False, type='str'),
             name=dict(required=False, type='str'),
+            description=dict(required=False, type='str'),
             data_id_map=dict(required=False, type='dict'),
+            tags=dict(required=False, type='dict'),
```

Comment:
yes, i spoke with @lucasponce. He said it's only a documentation error, and he will fix it.